text/069-coq-roadmap.md

```diff
@@ -0,0 +1,316 @@
+- Title: Coq roadmap
+
+- Drivers: Nicolas Tabareau (@tabareau), Théo Zimmermann (@Zimmi48)
+
+----
+
+# Summary
+
+This CEP aims to establish a roadmap for the Coq project.
+It highlights both:
+
+- what are considered as important axes to work on in the future,
```

Comment:
What I see so far is a list of things that people plan to work on.  This is a useful planning tool.  But there's really no discussion about the priorities, relative priorities or why something is considered a priority.  We should discuss the best ways to make Coq more useful to users without limiting it to things that people plan to work on in the near future.  For example, I think there are a lot of very useful improvements to make to GUIs such as vsCoq2 that require support from Coq.  Documentation needs a lots of work.

Seems like this should be a separate document and a different kind of discussion.

Reply:
We want to map the priorities in terms of need with the individual priorities in terms of volunteering effort. But the second section (better names welcome for the sections!) is precisely meant to also discuss priorities without committed volunteer effort (and then we can use this second section to ask Inria for engineering support).

Reply:
> We want to map the priorities in terms of need with the individual priorities in terms of volunteering effort.

Not entirely sure what you mean here.  There should be a place where we begin by stating user needs/priorities, then relating line items to the themes identified there and identifying which ones benefit users most and why.  The technical work should flow from user needs and priorities.  We're leaving them unstated and perhaps undiscussed.  But I think the team is not used to thinking this way.

Maybe we come back to this later.  If you like, I could provide a paragraph or two to fit into the document.

Reply:
>Maybe we come back to this later.

Yes, let's improve the process step by step.

> If you like, I could provide a paragraph or two to fit into the document.

Yes, sure. I would be interested in reading what you have in mind here.